src/main/java/com/github/jmchilton/blend4j/galaxy/WorkflowsClient.java

```diff
@@ -22,4 +22,11 @@
   ClientResponse runWorkflowResponse(WorkflowInputs workflowInputs);
 
   WorkflowOutputs runWorkflow(WorkflowInputs workflowInputs);
+  
+  /**
+   * Deletes the workflow with the given id.  This action is irreversible.
+   * @param id The id of the workflow to delete.
+   * @return A {@link ClientResponse} for this request.
+   */
+  ClientResponse deleteWorkflowResponse(String id);
```

Comment:
I'd rather avoid writing it since we aren't going to use it.  I did it for histories since I thought all responses looked the same, but it turns out they don't.  I didn't want to delete the code I already wrote and tested though.